302.go

```diff
@@ -942,7 +905,6 @@ func main() {
 				LoadConfig(*configPtr, *debugPtr)
 			case syscall.SIGUSR2:
 				logger.Infof("Got A USR2 Signal! Now Reopen log file....\n")
-				InitLoggers()
```

Comment:
This is for logrotate...

Reply:
This one is duplicated. `InitLoggers()` is called twice consecutively and the first error is even dropped. I only removed the first call.